xml/deployment_prep_aarch64_raspi.xml

```diff
@@ -101,105 +242,53 @@
-     or non-volatile memory to hold configuration information.
-    </para>
-    <para>
-     This means there are no conventional settings to adjust for boot device
-     order, time and date, and so on.
-    </para>
-    <para>
-     Instead, the &rpi; has a very small amount of One-Time Programmable Memory
-     (OTP memory) which can be used to configure some settings, such as whether
-     the machine should attempt to boot from USB devices or over Ethernet. It is
-     only possible to program this by preparing a bootable &microsd; card and
-     placing certain configuration files on it.
-    </para>
-    <para>
-     This OTP memory is described on the &rpi; Foundation Web site:
-     <link xlink:href="https://www.raspberrypi.org/documentation/hardware/raspberrypi/otpbits.mdd"/>
-    </para>
-    <note>
-     <title>Time Synchronization</title>
-     <para>
-      The lack of a Real Time Clock (RTC) also means that &rpi; devices need to
-      be configured to fetch the time from a network server by
-      &ntp;.
-    </para>
-    </note>
-   </listitem>
-   <listitem>
-    <para>
-     The primary processor on the &rpi; is the VideoCore IV Graphics Processing
-     Unit (GPU), not the &arm; processor. It is the GPU which initialises the
-     hardware and loads the operating system. As a result, most &rpi; operating
-     systems do not use a boot loader.
-    </para>
-    <para>
-     &sle; for the &rpi; is configured with a boot loader called
-     <literal>Das U-Boot</literal>, which is loaded before the operating system
-     and can optionally show a boot menu, like GRUB on &x86; hardware.
-    </para>
-   </listitem>
-  </orderedlist>
-
-  <para>
-   Installation can be performed from a bootable USB device, such as a memory
-   stick or USB optical drive. The exact procedure may differ from model to
-   model, depending on how the individual device is configured.
+   Newer models of &rpi; 3 with BCM2837 B0 silicon (silver instead of black chip),
+   including &rpi; 3 Model B+ and Compute Module 3+, allow to boot from USB-connected
+   storage devices by default.
   </para>
   <para>
-   The &rpi; 3 Model B+ is shipped pre-configured to boot from USB devices and
-   no additional steps should be required.
+   On older models, such as &rpi; 3 Model B or Compute Module 3, USB boot can
+   be enabled by booting from a specially prepared &microsd; card once.
+   Please see <xref linkend="sec-aarch64-deprpi-otp"/> for instructions.
   </para>
-  <para>
-   Depending on whether this has previously been done on a particular unit, the
-   older &rpi; 3 Model B may require additional configuration to boot from USB.
-   The procedure is described on the &rpi; Foundation Web site:
-   <link xlink:href="https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/msd.md"/>
-  </para>
-
  </sect2>
- <sect2>
+
+ <sect2 xml:id="sec-aarch64-deprpi-net">
   <title>
-   Network Booting the &rpi;
+   Installation from Network
   </title>
```

Comment:
This pre-existing multi-line title is causing whitespace artifacts, will collapse to one-line as the others.